src/components/IconLibrary/IconCategory.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { groupBy } from 'lodash';
```

Comment:
Super unrelated to the PR, but do we have something like `babel-plugin-lodash` setup for `lodash` for bundle size stuff? If not, should we just use the package directly? e.g. https://www.npmjs.com/package/lodash.groupby